src/main/webapp/app/exercises/text/assess/text-assessment-dashboard/text-assessment-dashboard.component.ts

```diff
@@ -109,6 +109,7 @@ export class TextAssessmentDashboardComponent implements OnInit {
             )
             .subscribe((submissions: TextSubmission[]) => {
                 this.submissions = submissions;
+                this.filteredSubmissions = this.submissions;
```

Comment:
This line is not necessary, the same is already done below.
```suggestion
```

Reply:
Done! 
Thank you!